test/looker/test_dynamic_client.rb

```diff
@@ -116,7 +116,7 @@ def verify(response, method, path, body='', query={}, content_type = nil)
     end
 
     it "invalid method name" do
-      mock = MiniTest::Mock.new.expect(:call, response){|env| confirm_env(env, method, path, body, query, content_type)}
+      mock = MiniTest::Mock.new.expect(:call, response){|env| confirm_env(env, :get, path, body, query, content_type)}
       sdk = sdk_client(default_swagger, mock)
```

Comment:
ug, I hate figuring out what needs to be mocked and when in this repo! In this case we don't need a mock at all because we're pre-loading the `default_swagger` so the client doesn't reach out for it

```suggestion
      sdk = sdk_client(default_swagger, nil)
```